Client/src/ArmoniK.DevelopmentKit.Client.Common/ArmoniK.DevelopmentKit.Client.Common.csproj

```diff
@@ -1,37 +1,24 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
-  <!-- <PropertyGroup>
-    <IsWindows Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::Windows)))' == 'true'">true</IsWindows>
-    <IsLinux Condition="'$([System.Runtime.InteropServices.RuntimeInformation]::IsOSPlatform($([System.Runtime.InteropServices.OSPlatform]::Linux)))' == 'true'">true</IsLinux>
-  </PropertyGroup> -->
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFrameworks>net472;net48;net5.0;net6.0</TargetFrameworks>
-    <DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
-    <DockerfileContext>..\..\..\..\..\..\..</DockerfileContext>
     <OutputType>Library</OutputType>
-    <Authors>$(Authors)</Authors>
-    <Company>ANEO</Company>
-    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
-    <IsPackable>true</IsPackable>
-    <PackageOutputPath>../../../publish</PackageOutputPath>
-    <RuntimeIdentifiers>win-x64;linux-x64</RuntimeIdentifiers>
-    <GenerateDocumentationFile>True</GenerateDocumentationFile>
-    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
-    <LangVersion>latest</LangVersion>
-    <SignAssembly>True</SignAssembly>
-    <AssemblyOriginatorKeyFile>../../../kp.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="ArmoniK.Api.Client" Version="3.0.2-SNAPSHOT.32.1a7a62f" />
+ <ItemGroup>
+    <PackageReference Include="ArmoniK.Api.Client" Version="$(ArmonikAPIVersion)" />
```

Comment:
Is visual studio able to update this package ?

Reply:
yes